TheraLang.BLL/Services/ProjectParticipationService.cs

```diff
@@ -69,6 +70,28 @@ public async Task<IEnumerable<ProjectParticipationDto>> GetAll()
             return projectParticipationDtos;
         }
 
+        public async Task<IEnumerable<ProjectParticipationDto>> GetProjectParticipations(int projectId)
+        {
+            var projectParticipations = await _unitOfWork.Repository<ProjectParticipation>()
+                .GetAll()
+                .Where(p=>p.ProjectId == projectId && p.Status == ProjectParticipationStatus.Approved)
+                .Include(p => p.User)
+                .Include(p => p.Project)
+                .ToListAsync();
+
+            var mapper = new MapperConfiguration(cfg => cfg.CreateMap<ProjectParticipation, ProjectParticipationDto>()
+                .ForMember(m => m.ProjectId, opt => opt.MapFrom(m => m.ProjectId))
```

Comment:
No need to specify if the name of fields the same

Reply:
done